PREREQUISITES.md

```diff
@@ -193,12 +193,34 @@ If disabled, enter BIOS and find the Intel SGX feature
 (it is usually under the "Advanced" or "Security" menu),
 enable Intel SGX, save your BIOS settings, and exit BIOS.
 
-Download and install libsgx-enclave-common version 2.7.101:
+Download and install libsgx-enclave-common version 2.10.100.2:
 ```bash
-wget https://download.01.org/intel-sgx/sgx-linux/2.7.1/distro/ubuntu18.04-server/libsgx-enclave-common_2.7.101.3-bionic1_amd64.deb
-sudo dpkg -i libsgx-enclave-common_2.7.101.3-bionic1_amd64.deb
+wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-enclave-common/libsgx-enclave-common_2.10.100.2-bionic1_amd64.deb
+sudo dpkg -i libsgx-enclave-common_2.10.100.2-bionic1_amd64.deb
 ```
 
+Download and install sgx-aesm-service package version 2.10.100.2:
+```bash
+wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/ubuntu18.04-server/debian_pkgs/utils/sgx-aesm-service/sgx-aesm-service_2.10.100.2-bionic1_amd64.deb
+sudo dpkg -i sgx-aesm-service_2.10.100.2-bionic1_amd64.deb
+```
+
+Install Intel SGX PSW debian packages from Intel SGX repository:
```

Comment:
Please mention that below instructions are from https://download.01.org/intel-sgx/sgx-linux/2.10/docs/Intel_SGX_Installation_Guide_Linux_2.10_Open_Source.pdf

Reply:
I would suggest instead of above comments we should ask the user to refer to documentation in above link. Document mentions installing components in following order

To install the driver, PSW, and SDK packages, you need the root (or sudo) privilege. Install the components in following order:
1. Intel® SGX driver
2. Intel® SGX PSW
3. Intel® SGX SDK

Reply:
Also if weinstall PSW as mentioned in SGX installation guide, then I don't think we need to separately install `sgx-aesm-service_2.10.100.2-bionic1_amd64.deb` and `libsgx-enclave-common_2.10.100.2-bionic1_amd64.deb` . Please check and confirm

Reply:
Installing SGX is difficult. Referring to a 20 page manual with many options does not make it easier.  We already do the same steps in the Docker files.

Reply:
> confirm

@manojgop with installation of psw packages mentioned in the guide, aesm service was not installed on the system. I had to manually install `libsgx-enclave-common_2.10.100.2-bionic1_amd64.deb` and `sgx-aesm-service_2.10.100.2-bionic1_amd64.deb` to bring up aesm service.

Reply:
So in updated PR, above packages are not installed

Reply:
That's correct